.github/workflows/release.yml

```diff
@@ -31,6 +32,6 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       with:
         upload_url: ${{ github.event.release.upload_url }}
-        asset_path: ${{github.workspace}}/classifai.zip
+        asset_path: ${{ github.workspace }}/classifai.zip
```

Comment:
lol is it this spacing that was breaking releases not including the classifai.zip file?

Reply:
I doubt it but I guess we'll see 🤷🏻